source/javascripts/components/_Workflow.js.erb

```diff
@@ -19,7 +19,7 @@ angular.module("BitriseWorkflowEditor").factory("Workflow", function($rootScope,
 
 			try {
 				step = stepSourceService.stepFromCVS(stepCVS);
-				step.userStepConfig = userStepConfig;
+				step.userStepConfig = _.extend(step.userStepConfig, userStepConfig);
```

Comment:
prevent `userStepConfig` override after load, so we could assign only `asset_urls` at load time, won't be removed by overwrite.